reset-pass/account-validator-tool/src/java/org/sakaiproject/accountvalidator/tool/otp/AcountValidationLocator.java

```diff
@@ -174,6 +185,8 @@ public ValidationAccount getAccount() {
 	public String validateAccount() {
 		log.debug("Validate Account");
 		
+		List<String> userReferences = new ArrayList<String>();
```

Comment:
Doesn't need changing but you can now use the diamond operator in Java 8. Eg: ` new ArrayList<>();`